MultiVersions/Forge/main/autumnity/lang/zh_tw.json

```
@@ -0,0 +1,214 @@
+  "block.autumnity.red_maple_leaf_carpet": "紅色楓木樹葉地毯",
+
+  "block.autumnity.maple_leaf_pile": "楓木落葉堆",
+  "block.autumnity.yellow_maple_leaf_pile": "黃色楓木落葉堆",
+  "block.autumnity.orange_maple_leaf_pile": "橙色楓木落葉堆",
+  "block.autumnity.red_maple_leaf_pile": "紅色楓木落葉堆",
+  
+  "item.autumnity.maple_boat": "楓木船",
+  "item.autumnity.maple_furnace_boat": "熔爐楓木",
+  "item.autumnity.maple_chest_boat": "儲物箱楓木船",
+  "item.autumnity.large_maple_boat": "大型楓木船",
+  
+  "item.autumnity.sap_bottle": "楓木樹液瓶",
+  "item.autumnity.syrup_bottle": "楓糖漿瓶",
+  "item.autumnity.foul_berries": "穢莓",
+  "item.autumnity.foul_berry_pips": "穢莓籽",
+  "item.autumnity.foul_soup": "穢莓湯",
+  "item.autumnity.pumpkin_bread": "南瓜麵包",
+  
+  "item.autumnity.snail_spawn_egg": "蝸牛 生怪蛋",
+  "item.autumnity.snail_shell_piece": "蝸牛殼碎片",
+  "item.autumnity.snail_shell_chestplate": "蝸牛殼胸甲",
+  
+  "item.autumnity.turkey_spawn_egg": "火雞 生怪蛋",
+  "item.autumnity.turkey_egg": "火雞蛋",
+  "item.autumnity.turkey_piece": "生火雞腿",
+  "item.autumnity.cooked_turkey_piece": "烤火雞腿",
+  
+  "item.autumnity.maple_leaf_banner_pattern": "旗幟圖案",
+  "item.autumnity.maple_leaf_banner_pattern.desc": "楓葉",
+  "item.autumnity.swirl_banner_pattern": "旗幟圖案",
+  "item.autumnity.swirl_banner_pattern.desc": "漩渦",
+  
+  "item.autumnity.snail_shell_chestplate.whenSneaking": "潛行時：",
+  
+  "entity.autumnity.boat": "船",
+  "entity.autumnity.snail": "蝸牛",
+  "entity.autumnity.turkey": "火雞",
+  "entity.autumnity.turkey_egg": "拋出的火雞蛋",
+  
+  "_comment": "This effect extends the duration of active potion effects.",
+  "effect.autumnity.extension": "延效",
+  "_comment": "When the effect is active eating food will replenish 50% more food points than normally.",
+  "effect.autumnity.foul_taste": "穢臭味道",
+  
+  "item.minecraft.potion.effect.extension": "延效藥水",
+  "item.minecraft.splash_potion.effect.extension": "飛濺 延效藥水",
+  "item.minecraft.lingering_potion.effect.extension": "滯留 延效藥水",
+  "item.minecraft.tipped_arrow.effect.extension": "延效之箭",
+  
+  "biome.autumnity.maple_forest": "楓木森林",
+  "biome.autumnity.maple_forest_hills": "楓木森林丘陵",
+  "biome.autumnity.pumpkin_fields": "南瓜地",
+  "biome.autumnity.yellow_spotted_forest": "黃楓點綴森林",
+  "biome.autumnity.orange_spotted_dark_forest": "橘楓點綴黑森林",
+  "biome.autumnity.red_spotted_taiga": "紅楓點綴葉林",
+  "biome.autumnity.boreal_maple_forest": "北方楓木森林",
+  "biome.autumnity.boreal_maple_forest_hills": "北方楓木森林丘陵",
+  
+  "subtitles.autumnity.entity.snail.eat": "蝸牛進食",
+  "subtitles.autumnity.entity.snail.hurt": "蝸牛受傷",
+  "subtitles.autumnity.entity.snail.death": "蝸牛死亡",
+  "subtitles.autumnity.entity.snail.step": "蝸牛爬行",
+  
+  "subtitles.autumnity.entity.turkey.ambient": "火雞咯叫聲",
+  "subtitles.autumnity.entity.turkey.aggro": "火雞憤怒地咯叫",
+  "subtitles.autumnity.entity.turkey.hurt": "火雞受傷",
+  "subtitles.autumnity.entity.turkey.death": "火雞死亡",
+  "subtitles.autumnity.entity.turkey.egg": "火雞下蛋",
+  
+  "subtitles.autumnity.block.turkey.cut": "火雞肉切塊",

```

> "subtitles.autumnity.block.turkey.cut": "Cutting turkey",
這個我找不到實際出現在哪（

驗證了一下，你可以把火雞肉放在世界上，然後用 Farmer's Delight 的菜刀把肉切成雞腿
```suggestion
  "subtitles.autumnity.block.turkey.cut": "切火雞肉",

```